src/color-chart/color-chart.js

```diff
@@ -101,7 +101,13 @@ const Self = class ColorChart extends ColorElement {
 		}
 
 		if (isFinite(min) && isFinite(max)) {
-			const axisData = getAxis(min, max, 10);
+			const axisData = getAxis({
+				min,
+				max,
+				initialSteps: 10,
+				rawMin: this[`${axis}Min`],
```

Comment:
What do we plan to do with the dataset min/max if we have user-provided ones? I don't think we should use them in any way, right? 

If so, I would frame it around `force`, rather than `raw`. 
Either as a two separate settings (`forceMin: true, forceMax: true`) or  one separate setting (`force: {min: true}` with `force: true` for both) or as part of min/max's value (`min: {min, force: true}`

Reply:
We would be passing them as `min` and `max` in that case, since we don't need the min and max of the actual dataset (do we?).

Reply:
Oh, right. Now I got it.

Thank you so much for your excellent feedback. I addressed it and am ready for another iteration.